_redirects.yml

```diff
@@ -596,3 +596,9 @@
   - /go/builders/selected/
 "https://www.docker.com/build-early-access-program/?utm_campaign=onboard-30-customer-zero&utm_medium=in-product-ad&utm_source=desktop_v4":
   - /go/build-eap/
+"/build/building/multi-platform/":
+  - /go/multi-platform/
+"/build/cache/backends/inline/":
+  - /go/inline-cache/
+"/build/exporters/oci-docker/":
+  - /go/docker-exporter/
```

Comment:
Should we prefix these with `build-` to avoid any overlapping with runtime?

Reply:
probably a good idea 👍🏻